.vsts-ci.yml

```diff
@@ -217,7 +217,8 @@ stages:
               _LinuxPortable: ''
               _RuntimeIdentifier: '--runtime-id linux-musl-x64'
               _BuildArchitecture: 'x64'
-              _AdditionalBuildParameters: '/p:OSName="linux-musl"'
+              # Pass in HostOSName when running on alpine
+              _AdditionalBuildParameters: '/p:OSName="linux-musl" /p:HostOSName="linux-musl"'
```

Comment:
Do you need to specify both `OSName` and `HostOSName` here?  Won't `OSName` default to `HostOSName` now if you don't specify it?

Reply:
yeah `OSName` can be removed as of my last change. Since this is blocking I will merge as is and make a separate PR to remove it.

Reply:
will be fixed as part of this: https://github.com/dotnet/installer/pull/10891